samples/f30-ostree.json

```diff
@@ -0,0 +1,109 @@
+{
+  "sources": {
+    "org.osbuild.dnf": {
```

Comment:
This is easier said than done, perhaps, but I think we should move away from using the dnf stage anywhere, so that we can drop it completely soon.

Reply:
Absolutely. Done, by actually ;)